.bazelignore

```diff
@@ -1,4 +1,2 @@
 docs
 examples
-examples/crate_universe
```

Comment:
Does this not need to stay?

Reply:
it's a sub-directory of `examples` so it should already be ignored. After quite a bit of testing with this change I've noticed no regressions but the documentation for [bazelignore](https://docs.bazel.build/versions/master/guide.html#bazelignore) is a bit lacking.